### ISSUE TITLE
Add database actions to actionlog

### ISSUE THREAD

**Issue by tienne-B** [TabbycatDebate/tabbycat#1870]
To have a single point of reference for actions on tournaments, rather than it being split in two based on where the action was taken. Would have to extend the admin.

**Comment by czlee** [comment]
Yeah, might also make more sense the other way round - to deprecate our custom `actionlog` module in favour of extending the Django's `LogEntry` thing. Or maybe change to a much better third-party activity log solution that can support this (if one exists). See also: #229
